__tests__/controllers/web/DateSearchController.spec.ts

```diff
@@ -75,7 +128,7 @@ describe('Deals with the Date Search Controller', () => {
       geographySearch: skipPath,
```

Comment:
The skip path is taking to the search results page instead of the geography screen.

Reply:
This might happen when the results set for date search is 0. Since geography search is a subset of date search, when there are no results for date search, we directly show the unhappy path instead of geography search.